src/utils/autoBotEngine.js

```diff
@@ -101,8 +101,9 @@ const addEmptyStep = (personality) => {
   };
 };
 
-const getFormattedOptionText = (step) =>
-  [step.text, ...step.options].join(DEFAULT_PERSONALITY_OPTION_SEPARATOR);
+// only return the text without the bullet points
+const getFormattedOptionText = (step) => step.text;
+// [step.text, ...step.options].join(DEFAULT_PERSONALITY_OPTION_SEPARATOR);
```

Comment:
Maybe say why you're commenting out these lines?

Reply:
I wanted to keep this line in case we wanted to revert back to displaying the bullet points ... As when a quick reply is selected, they go away. 

Maybe a solution could be to keep the quick reply buttons there but gray them out (disabling) them ?